src/test/scala/uk/gov/hmrc/ui/pages/dashboard/AddAProperty.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ui.pages.dashboard
+
+import uk.gov.hmrc.ui.pages.BasePage
+
+object AddAProperty extends BasePage {
+
+  def addAProperty(): Unit =
+    headerCheck("Add a property to your account")
+
+  def accountHomeLinkDisplay(link: String = "Account home"): Unit = {
```

Comment:
This def isn't needed since we will be checking navigation to dashboard via this link.